src/engine/sparqlExpressions/SparqlExpressionValueGetters.h

```diff
@@ -10,6 +10,7 @@
 #include "../../global/Id.h"
 #include "../ResultTable.h"
 #include "./SparqlExpressionTypes.h"
+#include "util/TypeTraits.h"
```

Comment:
Can you as a cleanup measure make the includes here absolute and consistent (not your fault, but your chance to fix).

Reply:
Done, see [dab0eb7](https://github.com/ad-freiburg/qlever/pull/1214/commits/dab0eb758c0545199311ecf664a47aa11a6a54f0).